Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -529,15 +537,18 @@ eval BackPressed state = do
                           else if state.props.isSaveFavourite then continueWithCmd state [pure $ (SaveFavouriteCardAction (SaveFavouriteCardController.OnClose))]
                           else if state.props.showShareAppPopUp then continue state{props{showShareAppPopUp=false}} 
                           else if state.props.showMultipleRideInfo then continue state{props{showMultipleRideInfo=false}}
+                          else if state.props.showLiveDashboard then do
```

Comment:
add a version check here for backward compatibility

Reply:
We check the version and hide the live_stats view so no need to add version check here as well